README.md

```diff
@@ -1,73 +1,48 @@
----
-services: active-directory
-platforms: ios
+--- 
+Services: active-directory
+platforms: iOS
 author: brandwe
+level: 100
+client: iOS Mobile App
+service: Microsoft Graph
+endpoint: AAD V2
 ---
 
-# An iOS task application with Azure AD v2.0 using third party libraries
-
-**NOTE regarding iOS 9:**
-
-Apple has released iOS 9 which includes support for App Transport Security (ATS). ATS restricts apps from accessing the internet unless they meet several security requirements incuding TLS 1.2 and SHA-256. While Microsoft's APIs support these standards some third party APIs and content delivery networks we use have yet to be upgraded. This means that any app that relies on Azure Active Directory or Microsoft Accounts will fail when compiled with iOS 9. For now our recommendation is to disable ATS, which reverts to iOS 8 functionality. Please refer to this [technote from Apple](https://developer.apple.com/library/prerelease/ios/technotes/App-Transport-Security-Technote/) for more informtaion.
-
-----
-
-Azure AD provides the Active Directory Authentication Library, or ADAL, for iOS clients that need to access protected resources.  ADAL’s sole purpose in life is to make it easy for your app to get access tokens.  To demonstrate just how easy it is, here we’ll build a Objective C To-Do List application that:
-
--	Gets access tokens for calling the Azure AD Graph API using the [OAuth 2.0 authentication protocol](https://msdn.microsoft.com/library/azure/dn645545.aspx).
--	Searches a directory for users with a given alias.
-
-To build the complete working application, you’ll need to:
-
-1. Register your application with Azure AD.
-2. Install & Configure ADAL.
-3. Use ADAL to get tokens from Azure AD.
-
-To get started, [download the completed sample](https://github.com/AzureADQuickStarts/NativeClient-iOS/archive/complete.zip).  You'll also need an Azure AD tenant in which you can create users and register an application.  If you don't already have a tenant, [learn how to get one](active-directory-howto-tenant.md).
+# An iOS Microsoft Graph API application with Azure AD v2.0 using third party libraries
 
```

Comment:
I recommend introducing the following links: Library source code, AAD Dev landing page, MSAL ObjC, and the v2 library doc.  Also, any reference docs the library exposes.